### ISSUE TITLE
Cannot install spacy due to regex distribution not found

### ISSUE THREAD

**Issue by nabsofken** [explosion/spaCy#3134]
> (venv) bash-3.2$ pip install spacy==2.0.18
Looking in indexes: https://pypi.python.org/pypi/
Collecting spacy==2.0.18
  Using cached https://files.pythonhosted.org/packages/bb/ab/b64e4aac5d762c9b8a3df5c2a2b2bc45d8c4314228f78972241ee3334cc0/spacy-2.0.18-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting regex==2018.01.10 (from spacy==2.0.18)
  Could not find a version that satisfies the requirement regex==2018.01.10 (from spacy==2.0.18) (from versions: 2018.11.22)
No matching distribution found for regex==2018.01.10 (from spacy==2.0.18)

Tried installing spacy but keep getting the regex package error. regex==2018.01.10 distribution does not exist upon checking. Can this be updated or is there a way to avoid this?

I'm on OSX Mojave and using Python 2.7.15

**Comment by ines** [comment]
Hmm, that's strange! It should be this version:
https://pypi.org/project/regex/2018.01.10/

Are you using the latest version of pip? And did you try clearing your pip cache? You can also try installing with `--no-cache-dir`.

**Comment by nabsofken** [comment]
Yes I have the latest pip (18.1) I did clear the cache folder (Library/Caches/pip/) and I also did a --no-cache-dir but im still getting no luck installing regex version 2018.01.10.
Btw I am running this inside PyCharm. I tried from the Terminal and I get a similar error but for the plac dist.

>   Could not find a version that satisfies the requirement plac<1.0.0,>=0.9.6 (from spacy) (from versions: 1.0.0)
No matching distribution found for plac<1.0.0,>=0.9.6 (from spacy)
